### ISSUE TITLE
[Feature Request] Supporting RSA keys on smartcards

### ISSUE THREAD

**Issue by cmonty14** [cyd01/KiTTY#120]
A similar [request](https://www.chiark.greenend.org.uk/~sgtatham/putty/wishlist/smartcard-auth.html) has been addressed to PuTTY in 2017, but this feature is still lacking.
Smartcards in this context are e.g. Yubikey, Nitrokey storing a OpenPGP key.

**Comment by lars18th** [comment]
Hi @cmonty14 ,

Already commented: https://github.com/cyd01/KiTTY/issues/72#issuecomment-558096834

**Comment by lars18th** [comment]
Please, @cmonty14  (or @cyd01 ) close this issue.

**Comment by lars18th** [comment]
Hi @ageis ,

> Why recommend yet another fork, and not just implement the relevant part of the [OpenPGP card spec](https://openpgpcard.cloudbook.wiki/resources/openpgp-card-3.3.pdf)? used by YubiKey and others?
> 
> PuTTy devs are seemingly not interested in this either. Their message is several years old; by now it's now "standardized" and also as someone who beta-tested hardware for Yubico, I can tell you there are plenty of users for this.

I'm not the developer of KiTTY (he is @cyd01 ), but I can explain my point of view:
- The Smartcard support is *required* ONLY in the Kageant agent.
- The KiTTY project is based on a lot of internal changes over PuTTY. So, any new functionality requires a lot of effort.
- Other different projects are focused on provide smartcard support to Pageant. And they do a good support.

So, why we need to add this funcionality to KiTTY? The `pageant.exe` binary can work in portable mode and it's fully compatible with KiTTY. So, you only need to copy any fork of it in your KiTTY subdirectoy. For example, in my case, I'm using "pageant-CAC" to use the keys in my cards... **with** KiTTY.

In summary, don't waste your time! Use another Pageant tool.
